src/NLog.Web/Internal/IHostEnvironment.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NLog.Web
```

Comment:
Can namespace be updated to NLog.Web.Internal

Reply:
All completed